spec/haskell/src/SEL4/Kernel/Thread.lhs

```diff
@@ -424,6 +427,10 @@ To switch to a new thread, we call the architecture-specific thread switch funct
 
 > switchToThread :: PPtr TCB -> Kernel ()
 > switchToThread thread = do
+>         runnable <- isRunnable thread
+>         assert runnable "thread must be runnable"
```

Comment:
For consistency: start with uppercase like the other assertions

Reply:
actually, there doesn't seem to be consistency, looks like old stuff all starts with uppercase, and new stuff with lowercase; @lsf37 thoughts?